### PR TITLE
Reorganizes and Adds Items to the Clothing Vendors

### PR DESCRIPTION
**Pull request by Herder05**
## What Does This PR Do

Changes Amount of Items in clothesmate from 1 to 2/3
2(Fedora,Overalls,waistcoat,strippedDress,SailorDress,RedEveningGown,BlackTango,IanShirt)
3(Classic Jeans, Camo Jeans, Black Jeans, Khaki Jeans, White Jeans, Red Pants, Black Pants, Tan pants, Blue pants, Track pants, miljackets,Scarfs,Ties)

Additions
-  Hydroponics Winter Coat - Hydrodrobe
- Black & Brown Trenchcoat - Clothesmate
- Black & Blue Jacket - Clothesmate
- Winter Coat - Clothesmate
- Black/Navy Cowboy Shirt - Autodrobe
- Winter Boots - Shoelord9000
- SolGov Cap - Hatlord9000
- Hair Flower Pin - Hatlord9000
- Black Beret - Hatlord9000
- Brown/Black Cowboy Hat- Hatlord9000

Items Moved

Autodrobe
- Black Skirt - Suitlord9000
- Jackboots - Shoelord 9000
- Olive/White/Navy/Desert Military Jacket - Clothesmate
- Blue/Red/Black Tie - Clothesmate
- Bandana & Black Bandana - Clothesmate

Clothesmate
- Laceup Shoes - Shoelord9000

Moves Glasses/Sunglasses above fingerless gloves in the clothesmate

Removals:

Clothesmate:
- Monocle, Present in Autodrobe and fits better
- Kilt, Present in Autodrobe and fits better
- Black Shoes, Present in shoelord or the floor
- Poncho & Sombrero, It and other variations already present in autodrobe
- Sandals, Present in Shoelord

Autodrobe:
- Waistcoat, Present in clothesmate fits that better

## Why It's Good For The Game

Gives Players more options when getting clothing because there is no reason that two people cant have a red scarf
Cleans up the locations of items in vendors so they look a bit nicer

## Testing

Ran the game
Checked the items were in the correct vendor and had a price

## Changelog
:cl:
tweak: moved things around in clothing vendors
/:cl:

### CODE REVIEW

**Comment by Herder05** [comment]
oh my god github why are you like this 
give me a bit to fix the syntax of the description

**Comment by Herder05** [comment]
Update 12/30/2022
Added
- Winter Coat
- Black/Navy Cowboy Shirt
- Black/Brown Cowboy Boots
- Winter Boots
- Sol Gov Cap
- Hair Flower pin
- Black Beret
- Brown/Black Cowboy hat

File: code/game/machinery/vendors/generic_vendors.dm [hunk 1]
```diff
@@ -120,13 +120,19 @@
...
+					/obj/item/clothing/head/beret/black = 10,
+					/obj/item/clothing/head/hairflower = 10)
 	contraband = list(/obj/item/clothing/head/bearpelt = 5)
 	premium = list(/obj/item/clothing/head/soft/rainbow = 1)
 	prices = list(/obj/item/clothing/head/bowlerhat = 20, /obj/item/clothing/head/beaverhat = 20, /obj/item/clothing/head/boaterhat = 20,
-				/obj/item/clothing/head/fedora = 20, /obj/item/clothing/head/beret = 20, /obj/item/clothing/head/fez = 20,
-				/obj/item/clothing/head/bearpelt = 30)
+				/obj/item/clothing/head/fedora = 20, /obj/item/clothing/head/beret = 20, /obj/item/clothing/head/beret/black = 20, /obj/item/clothing/head/fez = 20,
+				/obj/item/clothing/head/hairflower = 20, /obj/item/clothing/head/cowboyhat = 20, /obj/item/clothing/head/cowboyhat/black  = 20,
+				/obj/item/clothing/head/soft/solgov = 20,/obj/item/clothing/head/bearpelt = 30)
```
Comment (Sirryan2002) [inline code/game/machinery/vendors/generic_vendors.dm:135]:
```suggestion
				/obj/item/clothing/head/soft/solgov = 20, /obj/item/clothing/head/bearpelt = 30)
```

File: code/game/machinery/vendors/generic_vendors.dm [hunk 2]
```diff
@@ -120,13 +120,19 @@
...
+					/obj/item/clothing/head/beret = 10,
+					/obj/item/clothing/head/beret/black = 10,
+					/obj/item/clothing/head/hairflower = 10)
 	contraband = list(/obj/item/clothing/head/bearpelt = 5)
 	premium = list(/obj/item/clothing/head/soft/rainbow = 1)
 	prices = list(/obj/item/clothing/head/bowlerhat = 20, /obj/item/clothing/head/beaverhat = 20, /obj/item/clothing/head/boaterhat = 20,
-				/obj/item/clothing/head/fedora = 20, /obj/item/clothing/head/beret = 20, /obj/item/clothing/head/fez = 20,
-				/obj/item/clothing/head/bearpelt = 30)
+				/obj/item/clothing/head/fedora = 20, /obj/item/clothing/head/beret = 20, /obj/item/clothing/head/beret/black = 20, /obj/item/clothing/head/fez = 20,
+				/obj/item/clothing/head/hairflower = 20, /obj/item/clothing/head/cowboyhat = 20, /obj/item/clothing/head/cowboyhat/black  = 20,
```
Comment (Sirryan2002) [inline code/game/machinery/vendors/generic_vendors.dm:134]:
```suggestion
				/obj/item/clothing/head/hairflower = 20, /obj/item/clothing/head/cowboyhat = 20, /obj/item/clothing/head/cowboyhat/black = 20,
```

**Comment by Herder05** [comment]
im gonna be honest i dont know what the merge conflict was created by

**Comment by SteelSlayer** [comment]
@Herder05 the merge conflict is in the `code/game/machinery/vendors/generic_vendors.dm` file. I suggest resolving conflicts in VSC after updating your branch.

**Comment by Herder05** [comment]
that should hopefully fix i think i dont know i feel like i broke everything 
then i just went to the pr that caused the conflict and copy + pasted the code so i dont know anymore

**Comment by farie82** [comment]
Please deconflict so it can be merged

**Comment by farie82** [comment]
Closing the PR due to inactivity. You can reopen the PR at any later date if you wish.

**Pull request closed without merging**